docs/core.rst

```diff
@@ -63,6 +69,13 @@ Neural Potentials
     neuraldual.NeuralDualSolver
     neuraldual.NeuralDual
 
+Utilities
```

Comment:
maybe be more specific? Padding Utilities?

Reply:
Done.